### PR TITLE
Misc Test / List Pane dialog fixed

### PR DESCRIPTION
**Pull request by admich**
Added default method for: select-query and deselect-query.

Fixed :query-identifier in the accept calls and the setted variable.

Rewritten the formatting-table layout.

In the first list-pane only 10 items are visible and the scrollbars
are not viewable. This is a problem in list-pane implementation.

### CODE REVIEW

**Comment by admich** [comment]
This fix issue #448

File: Core/clim-core/dialog-views.lisp [hunk 1]
```diff
@@ -67,6 +67,16 @@ a query is selected for input. It is responsible for updating the
...
+(defmethod select-query (stream query record)
+  (declare (ignore stream query record))
+  nil)
+
+(defmethod deselect-query (stream query record)
+  (declare (ignore stream query record))
+  nil)
+
+
+
```
Comment (dkochmanski) [inline Core/clim-core/dialog-views.lisp]: one empty line between methods is enough

**Comment by dkochmanski** [comment]
after running demo and trying to close misc demo I have:

```
#<ACCEPTING-VALUES-RECORD X 0:775/2 Y 0:290  1 {100C6D52B3}> is not contained in #<CLIM-CLX::CLX-900-APPLICATION-PANE CLIM-DEMO::OUTPUT {100C246793}>.
   [Condition of type SIMPLE-ERROR]

Restarts:
 0: [ABORT] Return to application command loop.
 1: [ABORT] Return to application command loop.
 2: [ABORT] Return to application command loop.
 3: [RETRY] Retry SLIME REPL evaluation request.
 4: [*ABORT] Return to SLIME's top level.
 5: [ABORT] abort thread (#<THREAD "repl-thread" RUNNING {10018DFF93}>)

Backtrace:
  0: ((FLET "CONTINUATION5" :IN ERASE-OUTPUT-RECORD) #<CLIM-CLX::CLX-900-APPLICATION-PANE CLIM-DEMO::OUTPUT {100C246793}>)
  1: ((:METHOD INVOKE-WITH-OUTPUT-RECORDING-OPTIONS (OUTPUT-RECORDING-STREAM T T T)) #<CLIM-CLX::CLX-900-APPLICATION-PANE CLIM-DEMO::OUTPUT {100C246793}> #<CLOSURE (FLET "CONTINUATION5" :IN ERASE-OUTPUT-RE..
  2: ((:METHOD DO-GRAPHICS-WITH-OPTIONS-INTERNAL (MEDIUM T T)) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C2470F3}> #<CLIM-CLX:CLX-RENDER-MEDIUM {100C2470F3}> #<CLOSURE (FLET GRAPHICS-OP :IN INVOKE-WITH-DRAWING-OPTI..
  3: ((:METHOD DO-GRAPHICS-WITH-OPTIONS (MEDIUM T)) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C2470F3}> #<CLOSURE (FLET GRAPHICS-OP :IN INVOKE-WITH-DRAWING-OPTIONS) {7FBAA668FBAB}>) [fast-method]
  4: ((LABELS #:G5 :IN INVOKE-WITH-DRAWING-OPTIONS) #<CLIM-CLX:CLX-RENDER-MEDIUM {100C2470F3}>)
  5: ((:METHOD INVOKE-WITH-DRAWING-OPTIONS (SHEET T)) #<CLIM-CLX::CLX-900-APPLICATION-PANE CLIM-DEMO::OUTPUT {100C246793}> #<CLOSURE (FLET "GCONTINUATION9" :IN ERASE-OUTPUT-RECORD) {7FBAA668FCBB}>) [fast-m..
  6: ((:METHOD ERASE-OUTPUT-RECORD (T STANDARD-OUTPUT-RECORDING-STREAM)) #<ACCEPTING-VALUES-RECORD X 0:775/2 Y 0:290  1 {100C6D52B3}> #<CLIM-CLX::CLX-900-APPLICATION-PANE CLIM-DEMO::OUTPUT {100C246793}> T)..
```

File: Core/clim-core/dialog-views.lisp (hunk 1 above)
Comment (admich) [reply Core/clim-core/dialog-views.lisp]: lines removed

**Comment by admich** [comment]
I removed the empty lines.

I don't have your error.

**Comment by dkochmanski** [comment]
@nsrahmad could you check if you can reproduce the error I've mentioned? either way, I think we could merge this branch as is since it is an improvement over what we had anyway.

**Comment by nsrahmad** [comment]
@admich Reproducing that error is tricky.
1) Run the `Misc. Test` demo. Click on `List Pane Dialog`.
2) Click on `List Pane Dialog` **again**. Notice that the whole pane is cleared but not the "NT" of "VICE PRESIDENT".
3) Click on "Cancel". 
You will get the error.

**Comment by admich** [comment]
I think that the problem is that when a user click on a item on the list pane on the left, the pane on the right is erased and the accepting-values dialog is not closed properly.

It's possible to remove the error using the keyword `:own-window t ` in accepting-values calls. In this way the dialog pop-up in a different frame  and if you click twice you have two dialogs.

This is not an elegant solution for this apllication in which the tests are supposed to appear on the right pane.

Another possible solution is to remove `List Pane Dialog` test from `Misc. Test` and add it (with the right modification) to the `Accepting Value Test` where it is more appropriate.

Which solution do you prefer?

**Comment by admich** [comment]
@nsrahmad I made it with the last commit.

**Comment by nsrahmad** [comment]
LGTM

**Pull request merged**